package.json

```diff
@@ -64,6 +64,7 @@
     "ember-load-initializers": "^0.5.1",
     "ember-moment": "6.1.0",
     "ember-resolver": "^2.0.3",
+    "ember-string-ishtmlsafe-polyfill": "1.0.1",
```

Comment:
This should be listed in `dependencies` (not `devDependencies`).  If its in `devDependencies` it will only work when running in this repo's test suite (not when consumed without `npm link`).